spec/config/mongoid.rb

```diff
@@ -2,12 +2,14 @@
   config.connect_to('delayed_job_shallow_mongoid_test')
 end
 
+Mongo::Logger.logger = ::Logger.new('tmp/mongo.log') if defined?(Mongo)
```

Comment:
As of mongoid 5.0, it shows a lot of noisy logs in the test outout. With this line, the logger will send all the logs to `tmp/mongo.log`.